app/containers/RoomHeader/index.tsx

```diff
@@ -102,6 +120,8 @@ class RoomHeaderContainer extends Component<IRoomHeaderContainerProps, any> {
 			subtitle = subtitleProp;
 		}
 
+		const sourceT = sourceType ? { ...sourceType, connected } : undefined;
```

Comment:
This is awkward.
We already receive the data properly.
Why should we mutate it to add `connected`?
Why not just use `useSelector` to fetch `connected` inside the component?